### DISCUSSION TITLE
Тест

### DISCUSSION THREAD

**Discussion by aNNiMON** [TemporaryTeam/NoticEditor#88] [Team Posts]
На гитхабе завезли Team Discussions, го тестить

**Comment by SeTSeR** [comment]
Чего тестить, обсуждать всё равно нечего.

**Comment by NaikSoftware** [comment]
Согласен

**Comment by aNNiMON** [comment]
Из гитхаба вывезли Team Discussions, хватит тестить.

**Reply by SeTSeR** [reply]
А у меня в приложении ещё остались. Их перенесли куда-то?
